sources/platform/actors/development/actor_definition/output_schema.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: Output schema
+sidebar_position: 3
 description: Output schema is designed to help actor developers present the results to actor users in an attractive and comprehensive output UI.
-slug: /actors/development/output-schema
+slug: /actors/development/actor-definition/output-schema
 ---
 
-# Actor output schema
```

Comment:
I'm not convinced the examples with __EXAMPLE_SOMETHING are great. I also think you don't need to write the full definition for every single property anymore. E.g. the below can be just ignored AFAIK. You only need to list the overrides. And that should be documented somewhere, which those are.

```js
"___EXAMPLE_TEXT_FIELD___": {
                                "label": "Text",
                                "format": "text"
                            },
```

Reply:
Also, the text is not easy to read, e.g. below is confusing and easy to get lost in.
<img width="873" alt="CleanShot 2023-04-12 at 17 26 19@2x" src="https://user-images.githubusercontent.com/23726914/231505962-86fa5f03-503c-4168-81a9-a6685f128089.png">

Reply:
Overall, this would deserve a split, like Input Schema, and a rewrite for readability.

Reply:
Noted.

Reply:
This will be fixed in the next batch, it's still the original text.